### PR TITLE
Improve documentation of fullRobot example

### LINKED ISSUE #2663: Stop time of Modelica.Mechanics.MultiBody.Examples.Systems.RobotR3.fullRobot
The stop time in the experiment annotation of Modelica.Mechanics.MultiBody.Examples.Systems.RobotR3.fullRobot is set to 2 s, but documentation states 3 s as default stop time.

### PR DESCRIPTION
**Pull request by tobolar**
Closes #2663

### CODE REVIEW

File: Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo [hunk 1]
```diff
@@ -320,9 +320,16 @@ load inertia.
...
 Translate and simulate with the default settings
-(default simulation time = 3 s).
+(default simulation stop time = 2&nbsp;s).
+<p>
+</p>
+In the path planning block, there is incorporated a simulation termination condition.
+Thus, the simulation can be terminated before reaching the stop time.
+The condition depends on the joints' start and end positions, reference speeds
+and reference accelerations.
+For current settings, the simulation should be terminated at about 1.85&nbsp;s.
```
Comment (beutlich) [inline Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo]: I'd rather skip this last sentence.

File: Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo [hunk 2]
```diff
@@ -320,9 +320,16 @@ load inertia.
...
 This example animates a motion of a detailed model of the robot
 with predefined axes' angles over time.
-For animation CAD data is used.
+For animation, CAD data is used.
 Translate and simulate with the default settings
-(default simulation time = 3 s).
+(default simulation stop time = 2&nbsp;s).
+<p>
+</p>
+In the path planning block, there is incorporated a simulation termination condition.
```
Comment (beutlich) [inline Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo]: Or: The path planning block incorporates a simulation termination condition.

File: Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo [hunk 3]
```diff
@@ -320,9 +320,16 @@ load inertia.
...
+Thus, the simulation can be terminated before reaching the stop time.
+The condition depends on the joints' start and end positions, reference speeds
+and reference accelerations.
```
Comment (beutlich) [inline Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo]: Or: The condition depends on the start and end positions, reference speeds and reference accelerations of the joints.

File: Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo (hunk 1 above)
Comment (tobolar) [reply Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo]: I'm open to it. Just explain why.

Comment (beutlich) [reply Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo]: Because it is solver specific.

Comment (tobolar) [reply Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo]: > terminated at about

tries to let some uncertainity on the real termination time.
I would prefer to document that the used can expected the simulation termination before simulation stop time. May be the following text is more suitable:
_For current example settings, the termination condition should indeed be fulfiled right before  simulation stops._

**Pull request merged**